inc/languages/english/global.lang.php

```diff
@@ -268,6 +268,12 @@
 $l['postbit_warning_level'] = "Warning Level:";
 $l['postbit_warn'] = "Warn the author for this post";
 $l['postbit_purgespammer'] = "Purge Spammer";
+$l['postbit_post_deleted'] = "This post has been deleted.";
+$l['postbit_post_unapproved'] = "This post is awaiting approval.";
+$l['postbit_thread_deleted'] = "This thread has been deleted.";
+$l['postbit_thread_unapproved'] = "This thread is awaiting approval.";
+$l['postbit_deleted_post_user'] = "This post by {1} has been deleted.";
+$l['postbit_deleted_post'] = "This post has been deleted.";
```

Comment:
Is there a reason to add the same language string twice? `postbit_post_deleted` is the same